src/content/docs/custom-pipeline/setup.mdx

```diff
@@ -1,209 +1,186 @@
 ---
 title: Setting up your computer
-description: Setting up your computer for creating a custom pipeline.
+description: Setting up your computer to create a custom pipeline.
 ---
 
-## Prototype pipeline development container
+import CloneIcon from '~icons/codicon/source-control'
+import RemoteIcon from '~icons/codicon/remote'
+import ExtensionsIcon from '~icons/codicon/extensions'
+import CoffeeIcon from '~icons/codicon/coffee'
+import WaitIcon from '~icons/ic/round-access-time'
 
-## Requirements
+## Quick setup : VSCode and **devcontainers**
```

Comment:
This title is almost as big as the main title I think we should reduce the size of it.

Reply:
Played with **headings sizes**. It does affect the whole website though, but just changing heading doesn't look good.